config/intellij-idea-inspections.xml

```diff
@@ -1136,7 +1136,8 @@
                      enabled_by_default="true"/>
     <inspection_tool class="ClassOnlyUsedInOneModule" enabled="true" level="ERROR"
                      enabled_by_default="true"/>
-    <inspection_tool class="ClassOnlyUsedInOnePackage" enabled="true" level="ERROR"
+    <!-- packages should be where we define them. -->
```

Comment:
in library project ...... other projects might use it.

Reply:
Are you saying you want me to enable this but just use noinspection suppressions?

Reply:
Done.